pkgs/applications/misc/plover/default.nix

```diff
@@ -24,32 +29,78 @@
     ];
   };
 
-  dev = with python36Packages; mkDerivationWith buildPythonPackage rec {
+  dev = with python3Packages; mkDerivationWith buildPythonPackage rec {
     pname = "plover";
-    version = "4.0.0.dev8";
+    version = "4.0.0.dev9";
 
     meta = with lib; {
       description = "OpenSteno Plover stenography software";
       maintainers = with maintainers; [ twey kovirobi ];
-      license     = licenses.gpl2;
+      license     = licenses.gpl2Plus;
     };
 
-    src = fetchurl {
-      url    = "https://github.com/openstenoproject/plover/archive/v${version}.tar.gz";
-      sha256 = "1wxkmik1zyw5gqig5r0cas5v6f5408fbnximzw610rdisqy09rxp";
+    src = fetchFromGitHub {
+      owner = "openstenoproject";
+      repo = "plover";
+      rev = "54dbcf4ea73cc1ecc1d7c70dbe7bdb13f055d101";
```

Comment:
```suggestion
      rev = "v${version}";
```

Reply:
nope, they're doing a moving tag of "continuous" but still have a versioning scheme that's no longer integrated with git